### PR TITLE
contrib/wezterm: new package (20240203)

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

**Comment by ttyyls** [comment]
blocked on the same .cargo/config precedence issue as #1646

File: contrib/uthash/template.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+hostmakedepends = ["gmake", "perl"]
+pkgdesc = "C macros for hash tables and more"
+maintainer = "ttyyls <contact@behri.org>"
+license = "BSD-2-Clause"
+url = "https://troydhanson.github.io/uthash"
+source = (
+    f"https://github.com/troydhanson/uthash/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "e10382ab75518bad8319eb922ad04f907cb20cccb451a3aa980c9d005e661acc"
+options = ["empty"]
```
Comment (mhmdanas) [inline contrib/uthash/template.py:13]: a headers-only package doesn't need to have a -devel subpackage

```suggestion
```

File: contrib/uthash/template.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+def do_install(self):
+    self.install_license("LICENSE")
+    self.install_dir("usr/include")
+    self.install_file("src/*.h", "usr/include", glob=True)
+
+
+@subpackage("uthash-devel")
+def _devel(self):
+    return self.default_devel()
```
Comment (mhmdanas) [inline contrib/uthash/template.py:28]:
```suggestion
```

File: contrib/uthash/template.py (hunk 1 above)
Comment (ttyyls) [reply contrib/uthash/template.py:13]: sure

Comment (mhmdanas) [reply contrib/uthash/template.py:13]: you still need to remove this line :)

**Pull request closed without merging**